atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -453,45 +445,52 @@ private void loadWithTs(TableReference tableRef,
         }
         List<Callable<Void>> tasks = Lists.newArrayList();
         int fetchBatchCount = configManager.getConfig().fetchBatchCount();
-        for (final byte[] col : cellsByCol.keySet()) {
-            if (cellsByCol.get(col).size() > fetchBatchCount) {
+        for (Entry<byte[], SortedSet<Cell>> entry : Multimaps.asMap(cellsByCol).entrySet()) {
+            final byte[] col = entry.getKey();
+            SortedSet<Cell> columnCells = entry.getValue();
+            if (columnCells.size() > fetchBatchCount) {
```

Comment:
I've never really understood this warning.  Wouldn't us batching the thing according to the configured fetchBatchCount be the expected thing to do?  Is this just supposed to be saying "you're stupidly asking for too many things"?  I guess it does say that?  Is there a way to make this less noisy, basically if a product hits it once it's going to hit it a ton of times the the stacktraces blow up your logs.